lib/tmdb/movie.rb

```diff
@@ -0,0 +1,12 @@
+module Tmdb
+  class Movie
+    attr_reader :id, :original_title, :imdb_id, :release_date
+
+    def initialize(data = {})
+      @id = data['id']
```

Comment:
Should we use fetch to fail fast if some of the fields is not present in the data?

Reply:
As discussed already, let's keep it like this and validate later.